psutil/arch/windows/process_handles.c

```diff
@@ -244,7 +243,7 @@ psutil_get_open_files(DWORD dwPid, HANDLE hProcess) {
         }
 
         // This will set *globalFileName* global variable.
-        if (psutil_threaded_get_filename(hFile) != 0)
+        if (psutil_get_filename(hFile) != 0)
```

Comment:
This was not the cause of the weird error on Windows right? It should be removed